packages/generator/package.json

```diff
@@ -18,9 +18,9 @@
   ],
   "scripts": {
     "pretest": "npm run lint",
-    "test:debug": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\" --require ts-node/register --require source-map-support/register --timeout 5000 --colors",
-    "test": "nyc --extension .ts mocha --forbid-only \"test/**/*.test.ts\" --require ts-node/register --require source-map-support/register --reporter=xunit --reporter-options output=../../reports/generator.xml --colors",
-    "test:integration": "nyc --check-coverage=false --extension .ts mocha --forbid-only \"test_integration/**/*.ts\" --require ts-node/register --require source-map-support/register  --colors --reporter=xunit --reporter-options output=../../reports/generator-integration.xml",
+    "test:debug": "nyc mocha \"test/**/*.test.ts\"",
+    "test": "nyc mocha \"test/**/*.test.ts\" --reporter=xunit --reporter-options output=../../reports/generator.xml",
+    "test:integration": "nyc --check-coverage=false mocha \"test_integration/**/*.ts\" --reporter=xunit --reporter-options output=../../reports/generator-integration.xml --retries 2 --timeout 5000",
```

Comment:
[NIT] Since these options are the ones we're most likely to change and this is a very long line, can you move them to be the first two options?